sass/site/primary/_archives.scss

```diff
@@ -33,6 +33,12 @@
 	}
 }
 
+.archive .page-header .page-description {
+	display: block;
+	color: #111;
```

Comment:
This PR looks good, thanks @DannyCooper. Let's just swap this color value out with our Sass variable `$color__text-main`, and then we'll be ready to go. 👍